companies/tradedoubler.json

```diff
@@ -0,0 +1,40 @@
+{
+    "slug": "tradedoubler",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "ads"
+    ],
+    "name": "Tradedoubler AB",
+    "runs": [
+        "Tradedoubler GmbH",
+        "Adnologies",
+        "Interactive Marketing Works Ltd"
+    ],
+    "address": "Birger Jarlsgatan 57A\n113 56 - Stockholm\nSweden\n",
+    "phone": "+46 8 405 08 00",
+    "email": "privacy@tradedoubler.com",
+    "sources": [
```

Comment:
Please specify `https://www.tradedoubler.com` as the website.

Reply:
```suggestion
    "sources": [
        "https://www.tradedoubler.com",
```

Reply:
When I said "website", I meant the `website` property. :D I just commented on this line because that's where that property would go in the order given by the schema.